### PR TITLE
Stories Additional Code Removals

### PR DESCRIPTION
**Pull request by antonis**
Additional code removal suggestions on top of https://github.com/wordpress-mobile/WordPress-Android/pull/20016

-----

## To Test:

See https://github.com/wordpress-mobile/WordPress-Android/pull/20016

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

3. What automated tests I added (or what prevented me from doing so)

    - N/A

-----

## PR Submission Checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

-----

## Testing Checklist:

- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] Talkback.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] Large and small screen sizes. (Tablet and smaller phones)
- [ ] Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><code>strings.xml</code> files should only be updated on release branches, when the translations are downloaded by our automation.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">View files have been modified, but no screenshot or video is included in the pull request. Consider adding some for clarity.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 300 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WordPress/src/main/java/org/wordpress/android/ui/compose/components/buttons/ImageButton.kt [hunk 1]
```diff
@@ -39,10 +39,10 @@ fun PreviewDrawButton() {
...
                 shape = RoundedCornerShape(6.dp)
             ),
-        drawableLeft = Drawable(R.drawable.ic_story_icon_24dp),
-        drawableRight = Drawable(R.drawable.ic_story_icon_24dp),
-        drawableTop = Drawable(R.drawable.ic_story_icon_24dp),
-        drawableBottom = Drawable(R.drawable.ic_story_icon_24dp),
+        drawableLeft = Drawable(R.drawable.ic_pages_white_24dp),
+        drawableRight = Drawable(R.drawable.ic_pages_white_24dp),
+        drawableTop = Drawable(R.drawable.ic_pages_white_24dp),
+        drawableBottom = Drawable(R.drawable.ic_pages_white_24dp),
```
Comment (antonis) [inline WordPress/src/main/java/org/wordpress/android/ui/compose/components/buttons/ImageButton.kt:45]: I think any random icon would fit the purposes of the preview

File: WordPress/src/main/java/org/wordpress/android/ui/mysite/menu/MenuActivity.kt [hunk 2]
```diff
@@ -425,7 +425,7 @@ fun MySiteListItemPreviewWithSecondaryImage() {
         MenuItemState.MenuListItem(
             primaryIcon = R.drawable.ic_posts_white_24dp,
             primaryText = UiString.UiStringText("Plans"),
-            secondaryIcon = R.drawable.ic_story_icon_24dp,
+            secondaryIcon = R.drawable.ic_pages_white_24dp,
```
Comment (antonis) [inline WordPress/src/main/java/org/wordpress/android/ui/mysite/menu/MenuActivity.kt:428]: As above https://github.com/wordpress-mobile/WordPress-Android/pull/20341/files#r1508694602

**Review by jarvislin — APPROVED** [review]
Thank you for listing so many pieces of related code that I hadn't noticed. It looks fantastic.

**Pull request merged**